src/components/LoanCards/Buttons/LendAmountButton.vue

```diff
@@ -2,6 +2,7 @@
 	<lend-button
 		price="amountLeft"
 		:loan-id="loanId"
+		:is-partial-share="true"
```

Comment:
I suggest adding a new event here like so which also captures the smaller remaining amount value in the last param.
`v-kv-track-event="['Lending', 'Add to basket (Partial Share)', 'lend-button-click', loanId, amountLeft]"`

Reply:
ooh gotcha, yeah wasn't sure how `v-kv-track-event` works exactly and thought it was only tied to `kv-button` components for some reason. makes sense! 👍

Reply:
We should test to verify.

Alternatively, you could break out a new method, `addToBasket` which would replace `$emit('add-to-basket', $event)` in the listener and contain both the new tracking event and the original emit statement.

```
@add-to-basket="addToBasket($event)"
```

```
addToBasket(event) {
    this.$kvTrackEvent('Lending', 'Add to basket (Partial Share)', 'lend-button-click', this.loanId, this.amountLeft);
    $emit('add-to-basket', event);
}
```